lib/requireContent.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+  var global = this;
+  require.load = function (context, moduleName, url) {
+    var xhr;
+    xhr = new XMLHttpRequest();
+    xhr.open("GET", chrome.extension.getURL(url) + '?r=' + new Date().getTime(), true);
+    xhr.onreadystatechange = function (e) {
+      if (xhr.readyState === 4 && xhr.status === 200) {
+        eval.call(global, xhr.responseText + '\n//@ sourceURL=' + url);
+        context.completeLoad(moduleName)
+      }
+    };
+    xhr.send(null);
+  };
+})();
```

Comment:
Ah, interesting. I see: this is so that RequireJS works inside content scripts. Now I look closer, I can see that it only ever gets loaded inside the content scripts, and never in the normal app (where it won't work, because eval isn't allowed by default).

That makes sense, as is fine, but we should document it. I'd stick a comment in here referencing that link and explaining that this is needed because the normal requirejs loading mechanism doesn't work in content scripts, and I'd rename this file to something like "patch-require-for-content-scripts" so it's clearer what's going on at a quick glance. Otherwise this is fine :-)